node/src/daser.rs

```diff
@@ -499,8 +506,14 @@ mod tests {
     use celestia_types::test_utils::{generate_eds, ExtendedHeaderGenerator};
     use celestia_types::{AxisType, DataAvailabilityHeader, ExtendedDataSquare};
     use cid::Cid;
+    use std::collections::HashMap;
     use std::time::Duration;
 
+    // In which request number the tests can simulate invalid sampling
```

Comment:
```suggestion
    // Request number for which tests will simulate invalid sampling
```

Reply:
Also curious, why is this a `const` rather than plain variable passed in test?

Reply:
Because 2nd share will always be requested and it is a bit simpler to have a const than an argument.